.github/workflows/release-artifacts.yml

```diff
@@ -0,0 +1,86 @@
+name: Release artifacts
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
```

Comment:
I've tested this on the `push` trigger, but we don't have a good way to test the release creation from tag. Let's create the first tag after this is merged to test this CI job fully.